src/main/java/uk/gov/justice/hmpps/prison/api/resource/UserResource.java

```diff
@@ -71,7 +61,7 @@ public UserResource(final AuthenticationFacade authenticationFacade,
     @ProgrammaticAuthorisation("Returns information about the current user only")
     @GetMapping("/me")
     public UserDetail getMyUserInformation() {
-        return userService.getUserByUsername(authenticationFacade.getCurrentPrincipal());
+        return userService.getUserByUsername(hmppsAuthenticationHolder.getUsername());
```

Comment:
this is all about the current user information, so fine to replace with `getUsername`